src/mpid/ch4/src/ch4_init.c

```diff
@@ -91,9 +91,18 @@
         If the number is negative, underlying netmod or shmmod automatically uses an optimal number depending on
         the underlying fabric or shared memory architecture.
 
+    - name        : MPIR_CVAR_CH4_NUM_VCIS
```

Comment:
I think there should be different names for the CVAR and configure-time option. Both have `NUM_VCIS` as their suffix, which is confusing to the user.

Reply:
user don't see `MPIDI_CH4_NUM_VCIS`, they only see `--with-ch4-vcis=#` from `configure --help`. `MPIR_CVAR_....` is the only runtime variable user need know.

Reply:
In other word, config time and run-time to the user is totally separate, and there won't be chances for them to confuse.

Reply:
For developer, yes, there maybe confusion. All developer need remember is `MPIR_CVAR_xxx` are always runtime.

Reply:
Makes sense